grpcx/server.go

```diff
@@ -56,22 +58,71 @@ func WithRequestResponseLogs(l logx.Logger) grpc.UnaryServerInterceptor {
 
 // WithErrorLogs returns a gRPC interceptor for unary calls that instrument requests
 // with logs for the errors.
-func WithErrorLogs(l logx.Logger) grpc.UnaryServerInterceptor {
+func WithErrorLogs(l logx.Logger, options ...func(*withErrorLogsOptions)) grpc.UnaryServerInterceptor {
```

Comment:
the argument in the options func is not public, so callers won't be able to use them.

see an example impl here: https://github.com/socialpoint/jarvis-manager/blob/76351bbd7ab2b3d14533adf4f1d606f1b924eb67/pkg/apps/github/github.go#L69-L89

Reply:
👍   good catch! Changed!